authz/src/main/java/io/grpc/authz/AuthorizationServerInterceptor.java

```diff
@@ -31,14 +31,20 @@
 import java.util.List;
 
 /**
- * Authorization server interceptor for static policy.
+ * Authorization server interceptor for static policy. The class will get
+ * gRPC Authorization policy as a JSON string during initialization. This
+ * policy will be translated to Envoy RBAC policies and used to initialize
+ * GrpcAuthorizationEngine objects to make authorization decisions. It
+ * either contains two engines - deny engine followed by an allow engine
+ * or only one allow engine. This provider will use the same authorization
```

Comment:
"This provider" is weird, because there's no provider. And it is written from the internal perspective. How about, "The policy cannot be changed once created." (Then when we have the updating interceptor we could have a "To change the policy after creation, see TheOtherClass".)

Reply:
Agreed! I had removed "provider" in the last commit.
https://github.com/grpc/grpc-java/pull/8934/commits/fb79c7472d88407f6b4057558678be0b57d98e67
Updated the comment. Thank you for the suggestion:)